### PR TITLE
Pull request to add San Marcos High School bell schedule.

### PR DESCRIPTION
**Pull request by NicosHere**
I've followed the example and added as much detail as I could.

[Link to schedule](https://sanmarcoshigh.smusd.org/common/pages/UserFile.aspx?fileId=27852428)
[Link to Calendar](https://sanmarcoshigh.smusd.org/common/pages/UserFile.aspx?fileId=27852603)

Please email contactnicokatzen@gmail.com to contact me 👍

### CODE REVIEW

File: montavista/school.yml [hunk 1]
```diff
@@ -78,5 +78,5 @@ wednesday:
     - 15:20 Free
 
 weekend:
-  name: Weeekend
```
Comment (NicosHere) [inline montavista/school.yml:81]: While I was at it I fixed a typo with "Weeekend" changed it to "Weekend"

**Comment by arjunpat** [comment]
Thanks for submitting this pull request. For the most part, everything looks great!

I have one question: what is the difference between Period 0/7, Period 0, and Period 7.
Each of those will receive a box for students to enter custom period names. Is that the right thing to do considering your school's schedule?

**Comment by NicosHere** [comment]
Yes, because our school does not allow us to take a zero period and a seventh period so on monday's 0 period and 7th period are at the same time and that's why I added that in on top of the already existing 0 and 7

**Comment by NicosHere** [comment]
Do you need me to change anything?

**Comment by arjunpat** [comment]
Hi, thanks for the explanation!

I think it would be best to switch all "Period 0" to "Period 0/7" and all "Period 7" to "Period 0/7" because we don't want students to have to enter the same period twice.

Additionally, I don't think our current schedule validator allows for period names with slashes in them, but I can fix that as soon as we merge this pull request.

**Comment by NicosHere** [comment]
That couldn't work either because then all students with a 0/7 would have them twice in the schedule which would be wrong.

**Comment by arjunpat** [comment]
Ok. Then, let's remove "Period 0/7" from the periods section. That means that it will just show "Period 0/7" regardless on Mondays which I think is a fair balance between having people enter schedules twice and seeing "Period 0/7" twice. We do a similar thing for Mountain View High School on assembly days (we should "Period 2a" and "Period 2b" and don't let users customize that).

**Comment by NicosHere** [comment]
Done!

**Comment by arjunpat** [comment]
Awesome thanks! This will go live this weekend.

**Pull request merged**